packages/auth/lib/index.js

```diff
@@ -183,6 +188,20 @@ class FirebaseAuthModule extends FirebaseModule {
   }
 
   signInWithPhoneNumber(phoneNumber, forceResend) {
+    if (!isString(phoneNumber)) {
+      throw new FirebaseError(
```

Comment:
should this be wrapped in a `promise.reject()`?

Reply:
When invoking a Firebase method with invalid parameters, it should throw an exception immediately. Empty strings are not invalid parameters for the method, but simply values that fail to validate. This failure normally happens server-side, but the Java implementation throws uncaught native exceptions on empty strings, forcing us to mimick that locally instead with a `Promise.reject()`. This behaviour should mimick the regular Firebase behaviour as closely as possible.

Reply:
fair enough 👌